src/tests/python_tests.txt

```diff
@@ -44,7 +43,6 @@ test__osx_support   platform-specific
 test__xxsubinterpreters
 test_abc
 test_abstract_numbers
-test_aifc   audioop
```

Comment:
No I meant that we still need the `test_aifc`, just without the tag. Otherwise it wouldn't be run. Let me fit it.

Reply:
Got it, I had the mistaken impression that this was a list of exceptions.
Anyway, thanks for merging! Now I can discard my hack for importing `wave`. :smile: